### DISCUSSION TITLE
Can this sort of Node diagram be made in Freeplane? Specifcally with how "J" connects back to "B"

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1457] [Help]
Hey all. So I am trying to make a node diagram similar to this image I sketched out. Would something like this be possible to do in Freeplane? Thanks!

<img width="619" alt="abc" src="https://github.com/freeplane/freeplane/assets/147225942/b6f94f6f-ae1f-42bd-8041-0745838969dd">

**Comment by i-plasm** [comment]
Just use a connector from J to B:
![example-](https://github.com/freeplane/freeplane/assets/144830522/3c9a7962-0d27-4980-8c86-7d51156a4292)

**Reply by adamfreeplaneuser01** [reply]
Thanks, this is perfect!

**Comment by EdoFro** [comment]
You can also 

1. use Connector's Labels:

![image](https://github.com/freeplane/freeplane/assets/35700575/b78f114c-e851-44cd-8b2e-107eb6d4ba8a)

2. use summary node to group two or more nodes without inserting a parent to them (case nodes E and F inside an ellipse)

![image](https://github.com/freeplane/freeplane/assets/35700575/848d741c-b862-4971-9240-1da1c6bf85e4)

3. insert a parent node for E and F with a cloud and hide that node: cloud stays visible:

before hiding it:
![image](https://github.com/freeplane/freeplane/assets/35700575/3f900256-a12b-46a9-9387-0479b305cdb5)

after:
![image](https://github.com/freeplane/freeplane/assets/35700575/1ac2a599-f9ec-4b39-affa-607d0b085938)

**Reply by EdoFro** [reply]
I just learned that the correct form would be "to be hidden" 🤷🏼‍♂️

**Reply by adamfreeplaneuser01** [reply]
Besides making another node and hiding it, is there a way to get the cloud to cover multiple nodes? Sorry if that's a simple question lol
